elements/pl_file_editor/pl_file_editor.mustache

```diff
@@ -0,0 +1,25 @@
+{{#question}}
```

Comment:
`question_panel`?

Reply:
This is following a convention established somewhere of proxying the value of `data["panel"]` as a boolean in `html_params`. So in this case, `html_params["question"] == True` if the element is being rendered in a question panel.

Reply:
Ah! I had forgotten which level of templating we were in at the moment :-)